Gemfile.lock

```diff
@@ -248,10 +248,10 @@ GEM
     erubis (2.7.0)
     eventmachine (1.0.9.1)
     execjs (2.6.0)
-    factory_girl (4.7.0)
+    factory_girl (4.5.0)
       activesupport (>= 3.0.0)
-    factory_girl_rails (4.7.0)
-      factory_girl (~> 4.7.0)
+    factory_girl_rails (4.6.0)
+      factory_girl (~> 4.5.0)
       railties (>= 3.0.0)
```

Comment:
I think this was an inadvertent error due to merge conflicts. I think these 3 gems should stay at version 4.7.0.